recipes/mariadb-connector-c/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class MariadbConnectorcConan(ConanFile):
+    name = "mariadb-connector-c"
+    description = "MariaDB Connector/C is used to connect applications " \
+                  "developed in C/C++ to MariaDB and MySQL databases."
+    license = "LGPL-2.1-or-later"
+    topics = ("conan", "mariadb-connector-c", "mariadb", "mysql", "database")
+    homepage = "https://mariadb.com/kb/en/mariadb-connector-c"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "dyncol": [True, False],
+        "with_iconv": [True, False],
+        "with_curl": [True, False],
+        "with_ssl": [False, "openssl", "gnutls", "schannel"]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "dyncol": True,
+        "with_iconv": False,
+        "with_curl": True,
+        "with_ssl": "openssl"
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            del self.options.with_iconv
+            self.options.with_ssl = "schannel"
```

Comment:
You're assigning `schannel` to `self.otions.with_ssl`.
So I'm assuming it's ok to use another ssl library on Windows?

Reply:
Yes it's ok to use others ssl libraries on windows. It's just a conditional default option.

upstream default `WITH_SSL` value:
- windows: SCHANNEL
- non windows: OPENSSL

https://github.com/mariadb-corporation/mariadb-connector-c/blob/79137a4ae1cf37ab46940d26879051ad1dfce512/CMakeLists.txt#L67
https://github.com/mariadb-corporation/mariadb-connector-c/blob/79137a4ae1cf37ab46940d26879051ad1dfce512/CMakeLists.txt#L267

Reply:
Thans for the links.